src/pages/settings/InitialPage.js

```diff
@@ -103,6 +122,23 @@ const InitialSettingsPage = ({
     if (_.isEmpty(myPersonalDetails)) {
         return null;
     }
+
+    // Add free policies (workspaces) to the list of menu items
+    const workspaceMenuItems = _.chain(policies)
+        .filter(policy => (
+            !_.isEmpty(policy)
+            && policy.type === CONST.POLICY.TYPE.FREE
+            && policy.role === CONST.POLICY.ROLE.ADMIN
+        ))
+        .map(policy => ({
+            title: policy.name,
+            icon: Building,
+            action: () => Navigation.navigate(ROUTES.getWorkspaceRoute(policy.ID)),
+            emphasizeIcon: true,
+        }))
+        .value();
+    const menuItems = [].concat(workspaceMenuItems, defaultMenuItems);
```

Comment:
NAB, not sure if this is weird but we could also do 

```js
workspaceMenuItems.push(...defaultMenuItems);
```

instead of creating a new array